cime_config/config_grids.xml

```diff
@@ -3087,6 +3128,14 @@
       <map name="OCN2ATM_SMAPNAME">cpl/gridmaps/WCAtl12to45E2r4/map_WCAtl12to45E2r4_to_ne30pg2_mono.210318.nc</map>
     </gridmap>
 
+    <gridmap atm_grid="ne30np4.pg2" ocn_grid="SOwISC12to60E2r4">
+      <map name="ATM2OCN_FMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_SOwISC12to60E2r4_mono.210119.nc</map>
+      <map name="ATM2OCN_VMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_SOwISC12to60E2r4_bilin.210212.nc</map>
+      <map name="ATM2OCN_SMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_SOwISC12to60E2r4-nomask_bilin.210119.nc</map>
```

Comment:
Again, for my understanding, what's the distinction between a 'nomask' map and a normal map? Does the mask version mask out the area covered by the ice shelf?